__fixtures__/integration.js-and-rust-for-preview/.changes/config.json

```diff
@@ -2,11 +2,15 @@
   "pkgManagers": {
     "javascript": {
       "version": true,
-      "publish": "echo publishing ${ pkg.pkg } would happen here"
+      "publish": "echo publishing ${ pkg.tag ? '--tag pkg.tag' : '' } would happen here",
```

Comment:
`'--tag pkg.tag'` appears to only return a string. I think we want something like `'--tag ' + pkg.tag`. Same below.

Not a big deal, but there is an extra space if the template returns `''`.

Reply:
Ah I see. I fixed the fixture and getting the results we want:

```js
"javascript": {
  "publish": "echo publishing${ !pkg.tag ? ' --tag ' + pkg.tag : null } would happen here",
},
```